ui/hooks/useGasFeeInputs.js

```diff
@@ -412,6 +412,9 @@ export function useGasFeeInputs(
       if (networkAndAccountSupports1559) {
         estimatesUnavailableWarning = true;
       }
+      if (gasPriceToUse <= 0) {
```

Comment:
Hi, I'm trying to understand @danjm comment: `Gas price can be lower than 1 on custom networks`. If that's correct, that means a custom network can set gas price e.g. to 0, which should be allowed, but it seems this validation doesn't allow it.

Reply:
@dan437 My concern was for `0 < gasPrice < 1`

I don't think there is a case we need to support where `gasPrice === 0`

Reply:
For example, on optimism, gasPrice is (currently) always esimated to 0.015